boards/arm/nucleo_l452re/doc/index.rst

```diff
@@ -21,13 +27,18 @@ some highlights of the Nucleo L452RE board:
 - Three LEDs: USB communication (LD1), user LED (LD2), power LED (LD3)
 - One push-button: RESET
 
-.. image:: img/nucleo_l452re.jpg
-  :width: 250px
+.. image:: img/nucleo_l452re_p.jpg
+  :width: 363px
   :align: center
-  :height: 188px
-  :alt: Nucleo L452RE
+  :height: 408px
+  :alt: Nucleo L452RE-P
```

Comment:
Why replacing nucleo_l452re.jpg by nucleo_l452re_p.jpg ?
This directory includes both boards, so we should have both.
And also restore the binary itself (nucleo_l452re_p.jpg)

Reply:
I've just kept `nucleo_l452re_p.jpg` because the quality is far better (see [the current one](https://docs.zephyrproject.org/latest/_images/nucleo_l452re.jpg)). I don't think there's a need to keep both pictures in the repo because you won't notice the differences at first sight.

Reply:
I'm actually not a fan of having binary blobs (like images) inside a repo at all, as they bloat the repository size over time. A `git clone` of the Linux kernel is about 3 GB after almost 30 years of development. Zephyr is already at 472 MB after only 4 years. And this doesn't even include the modules and vendor HALs which are out of tree.

In that sense I would vote for just keeping the old image :)

Reply:
That I can understand. Please let me know what you prefer.

Reply:
I prefer reusing the existing one, but it's not a blocker for me, that's why I approved already.